client/src/helpers/APICalls/searchContest.ts

```diff
@@ -16,3 +20,15 @@ export async function fetchAllContestByUserId({ userId }: Props): Promise<Search
       error: { message: 'Unable to connect to server. Please try again' },
     }));
 }
+
+export async function fetchContestById({ contestID }: ContestProps): Promise<Contest> {
+  const fetchOptions: FetchOptions = {
+    method: 'GET',
+    credentials: 'include',
+  };
+  return await fetch(`/contest/${contestID}`, fetchOptions)
```

Comment:
Aren't this function and this one `fetchAllSubmissionByContestId` giving us the same response? If yes, then please delete the current one.

Reply:
On the dashboard there is a tab for contests created by the user "In progress" and "Completed". fetchAllContestByUserId fetches all the contest the user created by searching by userId, while fecthContestById fetches contest for the submission for users who are viewing the contest so it uses the contestId instead of the userId
![account](https://user-images.githubusercontent.com/33403043/127100575-9809f632-dd92-41c7-b537-f09fd5a0be89.png)

Reply:
Thanks for the clarification